minishift_sanity.py

```diff
@@ -69,13 +69,15 @@ def setUp(self):
         self.log.info("###########################################################################################")
         self.Hypervisor_Provider = self.params.get('Hypervisor_Provider')
         self.iso_url = self.params.get('iso_url')
-        self.Provisioning_OpenShift = self.params.get('Provisioning_OpenShift')
         self.Is_Downstream = self.params.get('Is_Downstream')
         self.RHN_Username = self.params.get('RHN_USERNAME')
         self.RHN_Password = self.params.get('RHN_PASSWORD')
-        sys.path.append(self.params.get('minishift_PATH'))
-        sys.path.append(self.params.get('Provisioning_OpenShift'))
         self.log.info("Is downstream: ", self.Is_Downstream)
+        oc_locations = glob.glob(os.environ['HOME'] + "/.minishift/cache/oc/v3.*/")
```

Comment:
This wont work in cygwin terminal because ```os.environ['HOME']``` returns cygwin home i.e ```/home/user_name``` but the oc path we need our code would be ```/cygdrive/c/Users/cdk/.minishift/cache/oc/v3***/```. It works on all platform except windows cygwin terminal.

So the expected code would be 
import platform 
self.platform = platform.system()
if "CYGWIN" in self.platform:
    homepath = os.getenv("USERPROFILE")
    winpath = '/' + homepath[:1].lower() + homepath[2:].replace("\\", "/")
    cygpath = "/cygdrive" + winpath +"/.minishift"
    oc_locations = glob.glob(cygpath +"/cache/oc/v3.*/")
else:
    homepath = os.getenv("HOME") +"/.minishift"
    oc_locations = glob.glob(homepath +"/cache/oc/v3.*/")
I have already tested the code with latest cdk 3.0 nightly build.